maraboupy/examples/0_NNetExample.py

```diff
@@ -5,21 +5,22 @@
 Top contributors (to current version):
   - Christopher Lazarus
   - Kyle Julian
-  
+
 This file is part of the Marabou project.
 Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
 in the top-level source directory) and their institutional affiliations.
 All rights reserved. See the file COPYING in the top-level source
 directory for licensing information.
 '''
 
+import os
+import pathlib
 import sys
 import numpy as np
 
 ## %
-# Path to Marabou folder if you did not export it
-
-# sys.path.append('/home/USER/git/Marabou')
+# Add path to Marabou folder if you did not export it
+sys.path.insert(0, os.path.join(str(pathlib.Path(__file__).parent.absolute()), "../../"))
```

Comment:
While I understand the motivation for this, I personally feel this may be setting users up for more pain. Are we going to get a lot of people saying "the examples file works just fine, but why doesn't my own file work?"? 

My own preference would be to have people getting a working installation sooner rather than later. Having said that, don't feel too strongly about it, so if you think is the best way happy to defer.

Reply:
This makes sense. I undid changes to the path but only fix the index issue.